recipe/meta.yaml

```
@@ -1,37 +1,49 @@
-{% set version = "3.0.1" %}
+{% set name = "uritemplate" %}
+{% set version = "4.1.1" %}
 
 package:
   name: uritemplate
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/u/uritemplate/uritemplate-{{ version }}.tar.gz
-  sha256: 5af8ad10cec94f215e3f48112de2022e1d5a37ed427fbd88652fa908f2ab7cae
+  url: https://github.com/python-hyper/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 64cae94edd83bbb0c2c49b15f2cb8192c3f8492af6bc468211d1e8b8496f5791
+  skip: True  # [py<39]
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - python
     - pip
+    - wheel
+    - setuptools
   run:
     - python
-    - simplejson >=2.5.0
 
 test:
+  source_files:
+    - tests
   imports:
     - uritemplate
+  commands:
+    - pip check
+    - pytest -v tests
+  requires:
+    - pip
+    - pytest
 
 about:
-  home: http://github.com/uri-templates/uritemplate-py/
-  license: Apache 2.0
+  home: https://uritemplate.readthedocs.org
+  license: Apache-2.0
   license_family: Apache
   license_file: LICENSE
-  summary: 'URI Templates'
-  dev_url: http://github.com/uri-templates/uritemplate-py/
+  summary: Implementation of RFC 6570 URI Templates
+  description: Simple python library to deal with URI Templates.
+  dev_url: https://github.com/python-hyper/uritemplate/tree/4.1.1
```

```suggestion
  dev_url: https://github.com/python-hyper/uritemplate/
```
No need to specify the version tag here.